magda-web-client/src/Components/RequestDataset/SuggestDataset.js

```diff
@@ -53,15 +56,26 @@ export default class Suggest extends React.Component {
         };
 
         return (
-            <RequestFormLogic
-                formProps={formProps}
-                alertProps={alertProps}
-                requestType="request"
-                senderName={this.state.senderName}
-                senderEmail={this.state.senderEmail}
-                message={this.state.message}
-                handleChange={this.handleChange}
-            />
+            <ReactDocumentTitle title={"Suggest a Dataset |" + config.appName}>
+                <div>
+                    <Breadcrumbs
+                        breadcrumbs={[
+                            <li>
+                                <span>Request a dataset</span>
```

Comment:
Given that the link and breadcrumb both say "Suggest a Dataset", this should probably match :).

Reply:
yup cool will do :)